### DISCUSSION TITLE
discord support invite link

### DISCUSSION THREAD

**Discussion by Matrex7** [pocketbase/pocketbase#3860] [Q&A]
i wanna join a official discord server to get help there if possible

**Comment by ganigeorgiev** [comment]
There is no official Discord support channel (_see similar discussion in https://github.com/pocketbase/pocketbase/discussions/548 with link to a community maintained one_).

The repo discussions are usually the recommended way to get support (or send an email to `support [at] pocketbase.io` if it is something private and cannot be shared publicly, eg. a security vulnerability or `pb_data` export).
